### PR TITLE
update readme for test kubernetes example

### PR DESCRIPTION
**Pull request by zpoint**
The current command in the doc doesn't work

```bash
ERROR: (gcloud.beta.container.clusters.create) ResponseError: code=400, message=Master version "1.29.1-gke.1589020" is unsupported.
ERROR: (gcloud.beta.container.clusters.create) ResponseError: code=400, message=Master version must be one of "REGULAR" channel supported versions [1.28.14-gke.1340000, 1.28.14-gke.1376000, 1.28.15-gke.1020000, 1.29.9-gke.1496000, 1.29.9-gke.1541000, 1.29.10-gke.1054000, 1.30.5-gke.1443001, 1.30.5-gke.1699000, 1.31.1-gke.1846000, 1.31.1-gke.2008000, 1.31.1-gke.2105000].
```

Official doc of [supported version](https://cloud.google.com/kubernetes-engine/docs/release-notes)

Fetch version from gcloud, then apply it in the command

Splace reformated by pre-commit

Tested (run the relevant ones):

- [ ] Code formatting: `bash format.sh`
- [x] Any manual or new tests for this PR (please specify below)
- [ ] All smoke tests: `pytest tests/test_smoke.py` 
- [ ] Relevant individual smoke tests: `pytest tests/test_smoke.py::test_fill_in_the_name` 
- [ ] Backward compatibility tests: `conda deactivate; bash -i tests/backward_compatibility_tests.sh`

### CODE REVIEW

File: tests/kubernetes/README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ curl --header "Content-Type: application/json-patch+json" \
...
...ww.googleapis.com/auth/devstorage.read_only","https://www.googleapis.com/auth/logging.write","https://www.googleapis.com/auth/monitoring","https://www.googleapis.com/auth/servicecontrol","https://www.googleapis.com/auth/service.management.readonly","https://www.googleapis.com/auth/trace.append" --num-nodes "2" --enable-autoupgrade --enable-autorepair --max-surge-upgrade 1 --max-unavailable-upgrade 0 --node-locations "us-central1-c" && gcloud beta container --project "${PROJECT_ID}" node-pools create "l4" --cluster "${CLUSTER_NAME}" --zone "us-central1-c" --machine-type "g2-standard-4" --accelerator "type=nvidia-l4,count=1" --image-type "COS_CONTAINERD" --disk-type "pd-balanced" --disk-size "100" --metadata disable-legacy-endpoints=true --scopes "https://www.googleapis.com/auth/devstorage.read_only","https://www.googleapis.com/auth/logging.write","https://www.googleapis.com/auth/monitoring","https://www.googleapis.com/auth/servicecontrol","https://www.googleapis.com/auth/service.management.readonly","https://www.googleapis.com/auth/trace.append" --num-nodes "2" --enable-autoupgrade --enable-autorepair --max-surge-upgrade 1 --max-unavailable-upgrade 0 --node-locations "us-central1-c"
```
Comment (romilbhardwaj) [inline tests/kubernetes/README.md:53]: The GKE versions keep changing every couple of weeks, and the changes in this line would likely stop working soon. To merge this PR, we should have a GKE_VERSION variable which automatically fetches a valid version to use.

Comment (zpoint) [reply tests/kubernetes/README.md]: Good catch

File: tests/kubernetes/README.md [hunk 2]
```diff
@@ -50,7 +50,13 @@ curl --header "Content-Type: application/json-patch+json" \
...
...s "https://www.googleapis.com/auth/devstorage.read_only","https://www.googleapis.com/auth/logging.write","https://www.googleapis.com/auth/monitoring","https://www.googleapis.com/auth/servicecontrol","https://www.googleapis.com/auth/service.management.readonly","https://www.googleapis.com/auth/trace.append" --num-nodes "2" --enable-autoupgrade --enable-autorepair --max-surge-upgrade 1 --max-unavailable-upgrade 0 --node-locations "${REGION}" && gcloud beta container --project "${PROJECT_ID}" node-pools create "l4" --cluster "${CLUSTER_NAME}" --zone "${REGION}" --machine-type "g2-standard-4" --accelerator "type=nvidia-l4,count=1" --image-type "COS_CONTAINERD" --disk-type "pd-balanced" --disk-size "100" --metadata disable-legacy-endpoints=true --scopes "https://www.googleapis.com/auth/devstorage.read_only","https://www.googleapis.com/auth/logging.write","https://www.googleapis.com/auth/monitoring","https://www.googleapis.com/auth/servicecontrol","https://www.googleapis.com/auth/service.management.readonly","https://www.googleapis.com/auth/trace.append" --num-nodes "2" --enable-autoupgrade --enable-autorepair --max-surge-upgrade 1 --max-unavailable-upgrade 0 --node-locations "${REGION}"
```
Comment (romilbhardwaj) [inline tests/kubernetes/README.md:59]: We probably need to update `--subnetwork` to also use `$REGION` instead of `us-central1`

**Review by romilbhardwaj — APPROVED** [review]
Thanks @zpoint!

**Pull request merged**